dfusion_rust_core/src/models/standing_order.rs

```diff
@@ -1,24 +1,41 @@
-use serde_derive::{Deserialize};
-use sha2::{Digest, Sha256};
-use web3::types::{H256, U256};
-use crate::models::{ConcatenatingHashable, RollingHashable};
 use crate::models;
+use crate::models::{ConcatenatingHashable, RollingHashable};
+
 use array_macro::array;
+use serde_derive::Deserialize;
+use sha2::{Digest, Sha256};
+use std::sync::Arc;
+use web3::types::{Log};
+use web3::types::{H256, U256};
+use graph::data::store::{Entity};
 
+use super::util::*;
 
 #[derive(Debug, Clone, Deserialize, Eq, Ord, PartialEq, PartialOrd)]
 #[serde(rename_all = "camelCase")]
 pub struct StandingOrder {
     pub account_id: u16,
     pub batch_index: U256,
-    orders: Vec<super::Order>,
+    pub valid_from_auction_id: U256,
```

Comment:
unrelated to this PR: can we make these non public?

Reply:
Now I see there's a `get_orders` method that borrows the orders. Is this a standard practice in rust?
I'll make it private again